packages/web/src/lib/web-mode.ts

```diff
@@ -242,19 +243,26 @@ export class WebMode {
 
 		// with logic signature we don't need signers.
 		const toBeSignedTxns = base64Txs.map((txn: string, txnId: number) => {
-			return execParams[txnId].sign === SignType.LogicSignature
-				? { txn: txn, signers: [] } // logic signature
-				: { txn: txn, authAddr: execParams[txnId].fromAccount?.addr }; // set signer
+			switch (execParams[txnId].sign) {
+				case SignType.LogicSignature:
+					return { txn: txn, signers: [] } // logic signature
+				case SignType.MultiSignature: {
+					const msig: MultiSig = execParams[txnId] as MultiSig;
+					return { txn: txn, msig: msig.mparams }; // multi singature
```

Comment:
I think we also need to pass signers. Also why dont we use `AppendSigToMultiSig` method?

Reply:
I have got the above structure from here: https://developer.algorand.org/docs/get-details/transactions/signatures/?from_query=multi%20sig#multisignatures.

I used the same structure and I am getting the expected response after signing from AlgoSigner.

Reply:
Please check the PR description image. It contains the response after signing from AlgoSigner. This is expected response.